Makefile

```diff
@@ -0,0 +1,27 @@
+# This rule runs the resource scaffolding script
+# $1 is the name of the resource to generate
+
+generate:
+	@cd ./scripts/generate_resource; \
+	go mod download; \
+	go mod tidy; \
+	go run . $(resource) ;
+
+vet:
+	go vet
+
+lint:
```

Comment:
what about `go fmt -c` ? https://github.com/hashicorp/terraform-provider-tfe/blob/main/scripts/gofmtcheck.sh

Reply:
:laughing: I didn't even consider this since I have `go fmt` running on save! Will add :+1: